sql/connect/common/src/main/protobuf/spark/connect/pipelines.proto

```diff
@@ -0,0 +1,142 @@
+syntax = "proto3";
+
+package spark.connect;
+
+import "spark/connect/relations.proto";
+import "spark/connect/types.proto";
+
+option java_multiple_files = true;
+option java_package = "org.apache.spark.connect.proto";
+
+// Dispatch object for pipelines commands.
+message PipelineCommand {
+  oneof command_type {
+    CreateDataflowGraph create_dataflow_graph = 1;
+    DefineDataset define_dataset = 2;
+    DefineFlow define_flow = 3;
+    DropDataflowGraph drop_dataflow_graph = 4;
+    StartRun start_run = 5;
+    StopRun stop_run = 6;
+    DefineSqlGraphElements define_sql_graph_elements = 7;
+  }
+
+  message DefineSqlGraphElements {
+    optional string dataflow_graph_id = 1;
+    optional string sql_file_name = 2;
```

Comment:
Something that occurred to me recently is that there could be SQL files with the same name in different subdirs. Should this be `sql_file_path`?

Reply:
I think this is a filepath in implementation, actually. Let me confirm.

Reply:
Where is this path pointing to?

Reply:
Changed to file_path. We'll rename this in the implementation too.

> Where is this path pointing to?

@hvanhovell this path is the local path to the SQL file. It's mostly used for disambiguation in our observability.